How can I report some issues about rowserrcheck linter?

Hello there,

I found a few cases where [rowserrcheck](https://github.com/jingyugao/rowserrcheck) reports false positive.
I'd like to report those issues but the `rowserrcheck` repository doesn't have enabled the Github issues.

How should I report them? Should I do it in this repository and mention the author?

I hit one false positive too. Following is a reproduction of the issue.

```go
package main

import (
	"database/sql"
	"fmt"
	"math/rand"

	_ "github.com/lib/pq"
)

func main() {
	db, err := sql.Open("postgres", "postgres://localhost:5432/postgres")
	if err != nil {
		panic(err)
	}
	defer db.Close()

	var rows *sql.Rows
	if rand.Float64() < 0.5 {
		rows, err = db.Query("select 1")
	} else {
		rows, err = db.Query("select 2")
	}
	if err != nil {
		panic(err)
	}
	defer rows.Close()
	for rows.Next() {
		fmt.Println("new rows")
	}
	if err := rows.Err(); err != nil {
		panic(err)
	}
}
```

It reports the following errors but actually, it's handled.

```sh
test.go:20:23: rows.Err must be checked (rowserrcheck)
		rows, err = db.Query("select 1")
		                    ^
test.go:22:23: rows.Err must be checked (rowserrcheck)
		rows, err = db.Query("select 2")
		                    ^
```

/cc @jingyugao

--- a/go.mod
+++ b/go.mod
@@ -14,7 +14,6 @@ require (
 	github.com/esimonov/ifshort v1.0.0
 	github.com/fatih/color v1.10.0
 	github.com/go-critic/go-critic v0.5.3
-	github.com/go-sql-driver/mysql v1.4.0
 	github.com/go-xmlfmt/xmlfmt v0.0.0-20191208150333-d5b6f63a941b
 	github.com/gofrs/flock v0.8.0
 	github.com/golangci/check v0.0.0-20180506172741-cfe4005ccda2
@@ -33,7 +32,6 @@ require (
 	github.com/jgautheron/goconst v0.0.0-20201117150253-ccae5bf973f3
 	github.com/jingyugao/rowserrcheck v0.0.0-20210130005344-c6a0c12dd98d
 	github.com/jirfag/go-printf-func-name v0.0.0-20191110105641-45db9963cdd3
-	github.com/jmoiron/sqlx v1.2.0
 	github.com/kulti/thelper v0.2.1
 	github.com/kunwardeep/paralleltest v1.0.2
 	github.com/kyoh86/exportloopref v0.1.8
@@ -71,7 +69,6 @@ require (
 	github.com/ultraware/whitespace v0.0.4
 	github.com/uudashr/gocognit v1.0.1
 	github.com/valyala/quicktemplate v1.6.3
-	golang.org/x/mod v0.4.0
 	golang.org/x/sys v0.0.0-20201009025420-dfb3f7c4e634 // indirect
 	golang.org/x/text v0.3.4 // indirect
 	golang.org/x/tools v0.0.0-20210105210202-9ed45478a130

--- a/go.mod
+++ b/go.mod
@@ -14,6 +14,7 @@ require (
 	github.com/esimonov/ifshort v1.0.0
 	github.com/fatih/color v1.10.0
 	github.com/go-critic/go-critic v0.5.3
+	github.com/go-sql-driver/mysql v1.4.0
 	github.com/go-xmlfmt/xmlfmt v0.0.0-20191208150333-d5b6f63a941b
 	github.com/gofrs/flock v0.8.0
 	github.com/golangci/check v0.0.0-20180506172741-cfe4005ccda2
@@ -30,8 +31,9 @@ require (
 	github.com/golangci/revgrep v0.0.0-20180526074752-d9c87f5ffaf0
 	github.com/golangci/unconvert v0.0.0-20180507085042-28b1c447d1f4
 	github.com/jgautheron/goconst v0.0.0-20201117150253-ccae5bf973f3
-	github.com/jingyugao/rowserrcheck v0.0.0-20191204022205-72ab7603b68a
+	github.com/jingyugao/rowserrcheck v0.0.0-20210130005344-c6a0c12dd98d
 	github.com/jirfag/go-printf-func-name v0.0.0-20191110105641-45db9963cdd3
+	github.com/jmoiron/sqlx v1.2.0
 	github.com/kulti/thelper v0.2.1
 	github.com/kunwardeep/paralleltest v1.0.2
 	github.com/kyoh86/exportloopref v0.1.8
@@ -69,6 +71,7 @@ require (
 	github.com/ultraware/whitespace v0.0.4
 	github.com/uudashr/gocognit v1.0.1
 	github.com/valyala/quicktemplate v1.6.3
+	golang.org/x/mod v0.4.0
 	golang.org/x/sys v0.0.0-20201009025420-dfb3f7c4e634 // indirect
 	golang.org/x/text v0.3.4 // indirect
 	golang.org/x/tools v0.0.0-20210105210202-9ed45478a130

--- a/test/testdata/rowserrcheck.go
+++ b/test/testdata/rowserrcheck.go
@@ -3,11 +3,63 @@ package testdata
 
 import (
 	"database/sql"
+	"fmt"
+	"math/rand"
 )
 
 func RowsErrNotChecked(db *sql.DB) {
 	rows, _ := db.Query("select id from tb") // ERROR "rows.Err must be checked"
 	for rows.Next() {
 
+	}
+}
+
+func issue943_1() {
+	db, err := sql.Open("postgres", "postgres://localhost:5432/postgres")
+	if err != nil {
+		panic(err)
+	}
+	defer db.Close()
+
+	var rows *sql.Rows
+	if rand.Float64() < 0.5 {
+		rows, err = db.Query("select 1")
+	} else {
+		rows, err = db.Query("select 2")
+	}
+	if err != nil {
+		panic(err)
+	}
+	defer rows.Close()
+	for rows.Next() {
+		fmt.Println("new rows")
+	}
+	if err := rows.Err(); err != nil {
+		panic(err)
+	}
+}
+
+func issue943_1() {
+	db, err := sql.Open("postgres", "postgres://localhost:5432/postgres")
+	if err != nil {
+		panic(err)
+	}
+	defer db.Close()
+
+	var rows *sql.Rows
+	if rand.Float64() < 0.5 {
+		rows, err = db.Query("select 1") // ERROR "rows.Err must be checked"
+	} else {
+		rows, err = db.Query("select 2") // ERROR "rows.Err must be checked"
+	}
+	if err != nil {
+		panic(err)
+	}
+	defer rows.Close()
+	for rows.Next() {
+		fmt.Println("new rows")
+	}
+	if err := rows.Err(); err != nil {
+		panic(err)
 	}
 }

--- a/test/testdata/rowserrcheck.go
+++ b/test/testdata/rowserrcheck.go
@@ -14,14 +14,10 @@ func RowsErrNotChecked(db *sql.DB) {
 	}
 }
 
-func issue943_1() {
-	db, err := sql.Open("postgres", "postgres://localhost:5432/postgres")
-	if err != nil {
-		panic(err)
-	}
-	defer db.Close()
-
+func issue943(db *sql.DB) {
 	var rows *sql.Rows
+	var err error
+
 	if rand.Float64() < 0.5 {
 		rows, err = db.Query("select 1")
 	} else {
@@ -30,35 +26,13 @@ func issue943_1() {
 	if err != nil {
 		panic(err)
 	}
+
 	defer rows.Close()
+
 	for rows.Next() {
 		fmt.Println("new rows")
 	}
-	if err := rows.Err(); err != nil {
-		panic(err)
-	}
-}
 
-func issue943_1() {
-	db, err := sql.Open("postgres", "postgres://localhost:5432/postgres")
-	if err != nil {
-		panic(err)
-	}
-	defer db.Close()
-
-	var rows *sql.Rows
-	if rand.Float64() < 0.5 {
-		rows, err = db.Query("select 1") // ERROR "rows.Err must be checked"
-	} else {
-		rows, err = db.Query("select 2") // ERROR "rows.Err must be checked"
-	}
-	if err != nil {
-		panic(err)
-	}
-	defer rows.Close()
-	for rows.Next() {
-		fmt.Println("new rows")
-	}
 	if err := rows.Err(); err != nil {
 		panic(err)
 	}

